test/liquid_handle_dispense_test.py

```diff
@@ -856,6 +856,166 @@ def test_tempest_bad_chip_param(self):
                 nozzle="abc",
             )
 
+    def test_mantis_default_params(self):
+        """Tests mantis default params"""
+        instruction = self.protocol.liquid_handle_dispense(
+            source=self.tube.well(0),
+            destination=self.flat.well(0),
+            volume="5:uL",
+            liquid=ProteinBuffer,
+        )
+
+        assert "mode_params" not in instruction.data
+
+        instruction = self.protocol.liquid_handle_dispense(
+            source=self.tube.well(0),
+            destination=self.flat.well(0),
+            volume="5:uL",
+            liquid=ProteinBuffer,
+            device="x_mantis",
+            model="high_volume",
+            diaphragm=0,
+            nozzle_size="0.1:mm",
+            tubing="LV",
+            z_drop="0.0:mm",
+            viscosity="1",
+        )
+
+        mode_params = {
+            "x_mantis": {
+                "model": "high_volume",
+                "diaphragm": 0,
+                "nozzle_size": "0.1:mm",
+                "tubing": "LV",
+                "z_drop": "0.0:mm",
+                "viscosity": "1",
+            }
+        }
+
+        assert instruction.data["mode_params"] == mode_params
```

Comment:
From PR 390: 

_can we also make assertions here about the shape of the LiHa? ie how many rows and columns would we dispense into with these params?_

Reply:
_similar note here, let's assert we have 1 row and 1 column in our mode params. and as a follow-up question, is it safe to assume/assert in the dispense method that if x_mantis is specified that we also default to 1 row and 1 column?_

Reply:
Added liha shape assertion to test in latest commit!